src/main/kotlin/org/neo4j/graphql/handler/QueryHandler.kt

```diff
@@ -34,7 +34,8 @@ class QueryHandler private constructor(
                 .argument(input(OFFSET, Scalars.GraphQLInt))
                 .type(GraphQLNonNull(GraphQLList(GraphQLNonNull(GraphQLTypeReference(type.name)))))
             if (orderingTypeName != null) {
-                builder.argument(input(ORDER_BY, GraphQLTypeReference(orderingTypeName)))
+                val orderType = GraphQLList(GraphQLNonNull(GraphQLTypeReference(orderingTypeName)))
```

Comment:
graphql automatically supports lists instead of single elements here for enums.

Reply:
Yes but we should define the schema correctly

Reply:
won't that break single element sorts then? that already exist?

Reply:
I tested it with GraphiQL and got no errors. And the tests are also green.
Can you point me to a doc describing the behavior you explained?